### ISSUE TITLE
scipy.optimize help browsing

### ISSUE THREAD

**Issue by rajarshitiwari** [spyder-ide/spyder#17442]
### What steps will reproduce the problem?

The error shows when clicking on any of the links in scipy.optimize help on the help panel

### Traceback
```python-traceback
Traceback (most recent call last):
  File "/home/rajarshi/miniconda3/envs/cupy/lib/python3.9/site-packages/spyder/plugins/help/widgets.py", line 876, in handle_link_clicks
    self.rich_text.load_url(url)
  File "/home/rajarshi/miniconda3/envs/cupy/lib/python3.9/site-packages/spyder/plugins/help/widgets.py", line 199, in load_url
    self.load(qurl)
AttributeError: 'RichText' object has no attribute 'load'
```

## Versions

* Spyder version: 5.1.5 
* Python version: 3.9.7
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Linux 5.13.0-30-generic

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.29.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.3.0 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.10.0 (OK)
pylint >=2.5.0;<2.10.0        :  2.9.6 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.2.2;<1.3.0          :  1.2.4 (OK)
pylsp_black >=1.0.0           :  None (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.1.0             :  5.1.1 (OK)
qtpy >=1.5.0                  :  1.10.0 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  58.0.4 (OK)
sphinx >=0.6.6                :  4.2.0 (OK)
spyder_kernels >=2.1.1;<2.2.0 :  2.1.3 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
xdg >=0.26                    :  0.27 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  None (OK)
matplotlib >=2.0.0            :  3.5.0 (OK)
numpy >=1.7                   :  1.22.2 (OK)
pandas >=1.1.1                :  None (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  1.9 (OK)
```

**Comment by ccordoba12** [comment]
Hey @rajarshitiwari, thanks for reporting. This error is already fixed in our latest version (5.2.2).

Since it's still not available in Anaconda, you need to create a new environment with conda-forge packages. For that, please close Spyder, open a Terminal and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

After that, you can connect Spyder to your base environment by following these instructions:

http://docs.spyder-ide.org/current/faq.html#using-existing-environment

**Comment by rajarshitiwari** [comment]
Hi @ccordoba12, thanks for the response! I will update spyder. Cheers!
